client/src/app/slides/motions/motion/motion-slide.component.ts

```diff
@@ -62,6 +62,13 @@ export class MotionSlideComponent extends BaseMotionSlideComponent<MotionSlideDa
 
     private _data: SlideData<MotionSlideData>;
 
+    /**
+     * Hide the pretext if it is longer than X characters
+     * (includes HTML code)
+     * After testing certain setups, this seems to be a good value
+     */
+    public maxPreTextLength = 2000;
```

Comment:
Is this value a potential config value? Whats about projector resolutions? Maybe you can scale the value with the width of the projector.